### PR TITLE
Pass extras and Biometric KYC sandbox photo

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

Pass extras and Biometric KYC sandbox photo

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

**Comment by jumaallan** [comment]
> Thanks @jumaallan ..do we need to make any updates to the documentation?

I will do that in the [docs](https://github.com/smileidentity/docs) repo @Yeboahmedia

**Comment by jumaallan** [comment]
> @jumaallan
> 
> 1. we need these extras named partnerParams to be consistent with the other server libraries and docs everywhere
> 2. Apart from user_id , job_id and jobtype these extras should be added to the partner params based on the key value pair and we should not have a hard coded string like photo

The photo key exists actually - https://docs.usesmileid.com/supported-id-types/for-individuals-kyc/backed-by-id-authority/test-data/customising-sandbox-test-data#_od59p7frph7i

Not sure if we need to move it to extras or not

**Comment by jumaallan** [comment]
> > @jumaallan
> > 
> > 1. we need these extras named partnerParams to be consistent with the other server libraries and docs everywhere
> > 2. Apart from user_id , job_id and jobtype these extras should be added to the partner params based on the key value pair and we should not have a hard coded string like photo
> 
> The photo key exists actually - https://docs.usesmileid.com/supported-id-types/for-individuals-kyc/backed-by-id-authority/test-data/customising-sandbox-test-data#_od59p7frph7i
> 
> Not sure if we need to move it to extras or not

Fixed

File: lib/src/main/java/com/smileidentity/compose/SmileIDExt.kt [hunk 1]
```diff
@@ -63,6 +66,7 @@ fun SmileID.SmartSelfieEnrollment(
     allowAgentMode: Boolean = false,
     showAttribution: Boolean = true,
     showInstructions: Boolean = true,
+    partnerParams: ImmutableMap<String, String> = persistentMapOf(),
```
Comment (vanshg) [inline lib/src/main/java/com/smileidentity/compose/SmileIDExt.kt:69]: I'm afraid this is going to error. We currently specify `implementation(libs.kotlin.immutable.collections)` in our build.gradle. For this to work without the partner updating anything, it needs to be changed to `api(libs.kotlin.immutable.collections)`

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
 ## 10.0.0-beta13 (unreleased)
 
 ### Added
+- Added `extras` as optional params on all job types
+- Added `idAuthorityBypassPhoto` on Sandbox BiometricKYC jobs
```
Comment (vanshg) [inline CHANGELOG.md:7]: I don't see `idAuthorityBypassPhoto` as a parameter anywhere
